src/ContentRepository/RepositoryTools.cs

```diff
@@ -368,19 +420,41 @@ public static string[] GetProtectedPaths(Content content)
             return permitted;
         }
 
+        /// <summary>
+        /// Returns the list of content-types that are allowed in the content-type of the requested content.
+        /// </summary>
+        /// <snCategory>ContentTypes</snCategory>
+        /// <param name="content"></param>
+        /// <returns>Content list of content-types.</returns>
         [ODataFunction]
         [AllowedRoles(N.R.Everyone)]
+        // ReSharper disable once InconsistentNaming
         public static IEnumerable<Content> GetAllowedChildTypesFromCTD(Content content)
         {
-            return content.ContentType.AllowedChildTypes.Select(ct => Content.Create(ct));
+            return content.ContentType.AllowedChildTypes.Select(Content.Create);
         }
 
         /// <summary>
-        /// Returns a path list containing items that have explicit security entry for Everyone group but does not have explicit security entry for Visitor user.
+        /// Returns a path list in the subtree of the requested content
+        /// containing items that have explicit security entry for Everyone group but
+        /// does not have explicit security entry for Visitor user.
         /// </summary>
-        /// <param name="root">Examination scope.</param>
-        /// <returns></returns>
+        /// <snCategory>Security</snCategory>
+        /// <remarks>
+        /// The result example:
+        /// <code>
+        /// [
+        ///   "/Root/(apps)/GenericContent/Versions",
+        ///   "/Root/(apps)/User/Logout",
+        ///   "/Root/Content",
+        ///   "/Root/Trash"
+        /// ]
+        /// </code>
+        /// </remarks>
+        /// <param name="root"></param>
+        /// <returns>Path list.</returns>
         [ODataFunction]
+        [AllowedRoles(N.R.Everyone)]
```

Comment:
I think this should be
```suggestion
        [AllowedRoles(N.R.Administrators, N.R.Developers)]
```